pages/settings/index.html

```diff
@@ -288,7 +288,14 @@ <h3 data-string="settingsKeyboardShortcutsHeading"></h3>
         <ul id="key-map-list"></ul>
       </div>
     </div>
-
+    <div class="settings-container" id="custom-bangs-container">
+        <h3 data-string="settingsCustomBangs"></h3>
+        <div class="setting-section">
+            <button data-string="settingsCustomBangsAdd" id="add-custom-bang"></button>
```

Comment:
I think the button should probably go underneath the list.

Reply:
okay